wiki/content/deploy/index.md

```diff
@@ -1641,22 +1641,55 @@ The following configuration options are available for Alpha:
 * `--tls_client_auth string` - TLS client authentication used to validate client connection. See [Client authentication](#client-authentication) for details.
 
 ```sh
-# Default use for enabling TLS server (after generating certificates)
+# First, create rootca and node certificates and private keys
+$ dgraph cert -n localhost
+# Default use for enabling TLS server (after generating certificates and private keys)
 $ dgraph alpha --tls_dir tls
 ```
 
 Dgraph Live Loader can be configured with following options:
 
-* `--tls_dir string` - TLS dir path; this enables TLS connections (usually 'tls').
+* `--tls_cacert string` - Dgraph Root CA, such as `./tls/ca.crt`
 * `--tls_use_system_ca` - Include System CA with Dgraph Root CA.
 * `--tls_server_name string` - Server name, used for validating the server's TLS host name.
 
 ```sh
-# First, create a client certificate for live loader. This will create 'tls/client.live.crt'
-$ dgraph cert -c live
-
 # Now, connect to server using TLS
-$ dgraph live --tls_dir tls -s 21million.schema -f 21million.rdf.gz
+$ dgraph live --tls_cacert ./tls/ca.crt --tls_server_name "localhost" -s 21million.schema -f 21million.rdf.gz
+```
+
+### mTLS (Mutual TLS) options
+
+The following configuration options are available for Alpha:
+
+* `--tls_dir string` - TLS dir path; this enables TLS connections (usually 'tls').
+* `--tls_use_system_ca` - Include System CA with Dgraph Root CA.
+* `--tls_client_auth string` - TLS client authentication used to validate client connection. mTLS will require either `REQUIREANY` or `REQUIREANDVERIFY` for this setting. See [Client authentication](#client-authentication) for details.
```

Comment:
1. connection -> certificate
2. Lets not say, `"mTLS` will require ....`
All 4 options are valid for mTLS from least secure to most secure.

Reply:
Two options do not require client cert/key.  If client cert/key are not use, then this is not mTLS?

Reply:
mTLS is a loose term. 

In all 4 options, the server request the Client-Cert.
The other 2 options allows Client to optionally send Client Certificate.

Best to stray away for saying just 2 options are mTLS.

Reply:
mTLS is a loose term. Is it when server requests Client-Cert? or when Client send certificate? or when Client cert is verified?

In all 4 options, Server requests the client-cert. And in some cases client may or may not send client-cert and it may or may not be verified. 
Best to stray away from this.

Reply:
Thank you for explanation.